extensions/wikia/ARecoveryEngine/ResourceLoaders/ResourceLoaderAdEngineSourcePointMMSClient.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+class ResourceLoaderAdEngineSourcePointMMSClient extends ResourceLoaderAdEngineSourcePointBase {
+	const CACHE_BUSTER = 0;     // increase this any time the local files change
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)